cves/kernel/CVE-2022-36879.yml

```diff
@@ -186,8 +186,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: There is a lack of mention of a violation specification
```

Comment:
Places/websites checked to look for violations should have been mentioned. E.g. "mailing lists, bug reports"

Reply:
thanks! I updated the field.